drizzlepac/hlautils/catalog_utils.py

```diff
@@ -613,43 +629,35 @@ def measure_sources(self, filter_name):
-                            "MERR_{}".format(self.aper_radius_arcsec[1]): '6.3f', "MSKY": '10.8f', "STDEV": '10.8f',
-                            "FLUX_{}".format(self.aper_radius_arcsec[1]): '10.8f', "CI": "7.3f"}
+        final_col_format = {"X-Center": "18.13f", "Y-Center": "18.13f", "RA": "13.10f", "DEC": "13.10f", "ID": ".8g", "MagAp1": '6.3f', "MagErrAp1": '6.3f', "MagAp2": '6.3f',
+                            "MagErrAp2": '6.3f', "MSkyAp2": '10.8f', "StdevAp2": '10.4f',
+                            "FluxAp2": '10.8f', "CI": "7.3f", "Flags": "3d"} # TODO: Standardize precision
         for fcf_key in final_col_format.keys():
             output_photometry_table[fcf_key].format = final_col_format[fcf_key]
 
-        # change some column titles to match old daophot.txt files
-        rename_dict = {"XCENTER": "X-Center", "YCENTER": "Y-Center",
-                       "MAG_{}".format(self.aper_radius_arcsec[0]): "MagAp({})".format(self.aper_radius_arcsec[0]),
-                       "MAG_{}".format(self.aper_radius_arcsec[1]): "MagAp({})".format(self.aper_radius_arcsec[1]),
-                       "MERR_{}".format(self.aper_radius_arcsec[0]): "MagErr({})".format(self.aper_radius_arcsec[0]),
-                       "MERR_{}".format(self.aper_radius_arcsec[1]): "MagErr({})".format(self.aper_radius_arcsec[1]),
-                       "MSKY": "MSky({})".format(self.aper_radius_arcsec[1]),
-                       "STDEV": "Stdev({})".format(self.aper_radius_arcsec[1]),
-                       "FLUX_{}".format(self.aper_radius_arcsec[1]): "Flux({})".format(self.aper_radius_arcsec[1])}
-        for old_col_title in rename_dict:
-            output_photometry_table.rename_column(old_col_title, rename_dict[old_col_title])
-            log.info("Column '{}' renamed '{}'".format(old_col_title, rename_dict[old_col_title]))
+        # add units to columns
+        final_col_units = {"X-Center": "Pixels", "Y-Center": "Pixels", "RA": "Sky Coords", "DEC": "Sky Coords",
+                           "ID": "Unitless", "MagAp1": "ABMAG", "MagErrAp1": "ABMAG", "MagAp2": "ABMAG",
+                           "MagErrAp2": "ABMAG", "MSkyAp2": "ABMAG", "StdevAp2": "ABMAG",
+                           "FluxAp2": "erg cm^-2 sec^-1 Hz^-1", "CI": "ABMAG", "Flags": "Unitless"}
+        for col_title in final_col_units:
+            output_photometry_table[col_title].unit = final_col_units[col_title]
 
         # Capture specified columns in order to append to the total detection table
-        magap_name = "MagAp({})".format(self.aper_radius_arcsec[1])
-        self.subset_filter_source_cat = output_photometry_table["ID", "CI", magap_name, "Flags"]
-        self.subset_filter_source_cat.rename_column(magap_name, magap_name + "_" + filter_name)
+        self.subset_filter_source_cat = output_photometry_table["ID", "RA", "DEC", "MagAp2", "CI", "Flags"]
+        self.subset_filter_source_cat.rename_column("MagAp2", "MagAP2_" + filter_name)
```

Comment:
I realize the POINT catalogs are generated in a different manner than the SEGMENT catalogs.  However, I am not sure why the RA and DEC columns are being pulled from the filter catalog and being appended to the total catalog. Are the original total catalog RA/DEC the same as the RA/DEC obtained from the measurements or ???

Reply:
yes. The original total catalog X and Y are what's being used by measure_sources() to produce the filter catalogs. RA and DEC are computed based on these X and Y values, so the RA and DEC from the total catalog are the same as the filter RA and DEC.